processor/error/payload.go

```diff
@@ -15,6 +16,8 @@ type payload struct {
 func (pa *payload) transform() []beat.Event {
 	var events []beat.Event
 
+	logp.Debug("error", "Transform error events: events=%d, app=%s, agent=%s:%s", len(pa.Events), pa.App.Name, pa.App.Agent.Name, pa.App.Agent.Version)
```

Comment:
any particular reason you went with `name:version`? We used to do `name/version`